doc/src/learn/exchange-integration-guide.md

```diff
@@ -0,0 +1,238 @@
+---
+title: Sui Exchange Integration Guide
+---
+
+This topic describes how to integrate SUI, the token native to the Sui network, into a cryptocurrency exchange. The specific requirements and processes to implement an integration vary between exchanges. Rather than provide a step-by-step guide, this topic provides information about the primary tasks necessary to complete an integration. After the guidance about how to configure an integration, you can also find information and code samples related to staking and delegation on the Sui network.
+
+## Requirements to configure a SUI integration
+
+The requirements to configure a SUI integration include:
+ * A Sui Full node. You can operate your own Sui Full node or use a Full node from a node operator.
+ * Suggested hardware requirements to run a Sui Full node:
+    * CPU: 10 core
+    * RAM: 32 GB
+    * Storage: 1 TB SSD
+
+We recommend running Sui Full nodes on Linux. Sui supports the Ubuntu and Debian distributions.
+
+## Configure a Sui Full node
```

Comment:
Should we link to `src/build/fullnode` here to make sure these instructions stay in sync?

Reply:
Yes, eventually. That topic is not specific to this use case. The new doc site supports reusable text blocks, so we can include the same content in multiple places from a single source. We'll convert this to use that mechanism when we move it over. The Docker stuff is currently only in a readme, so adding it to docs